docs/pages/teleport-editions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Teleport Editions
+description: Provides a comparison of Teleport Enterprise (Self-Hosted), Teleport Enterprise (Cloud), and Teleport Community Edition.
+---
+
+This guide explains the editions and products that are available to Teleport
+users.
+
+Teleport includes two editions:
+- **Teleport Community Edition:** An open source offering intended for demos and
+  small teams.
+- **Teleport Enterprise:** A fully-featured commercial offering.
+
+Teleport Enterprise offers two deployment
+options: 
+- **Cloud:** The Teleport team manages the Teleport Auth Service and Teleport
+  Proxy Service on the Teleport Cloud infrastructure.
+- **Self-Hosted:** Teleport users deploy the Teleport Auth Service and Teleport Proxy
+  Service on their own infrastructure.
+
+Teleport Enterprise includes add-on products that provide a more complete
+infrastructure identity solution, which this guide explains in more detail
+below.
+
+For information on how Teleport calculates billing metrics for product usage,
+see [Usage and Billing](./usage-billing.mdx).
+
+## Teleport Zero Trust Access
+
+Teleport Zero Trust Access provides engineers with just-in-time,
```

Comment:
The _just-in-time_ part refers to Access Requests (I assume), which is part of Teleport Identity Governance, not part of Zero Trust Access.

Reply:
I copied this paragraph from the Zero Trust Access messaging guide ([Google Drive](https://docs.google.com/document/d/15yNRcMQJuwIeeoouzF1CeVwg8ifcCACj76XWB0CN7Pc/edit?tab=t.0)), but I agree that it's incorrect. I'll fix.